Reader.md

```diff
@@ -0,0 +1,176 @@
+# Intro
+
+Welcome to the series of videos about functional programming in Kotlin with Arrow. Arrow is a library that is packed with data types and type classes bringing typed FP to Kotlin. In this video, we're going to learn about the Reader type and what it's used for.
+
+# Slide 1
+```
+Reader
+
+Reader is a higher kinded wrapper around a function. This function will be ran at some point in the future, when we can provide a proper execution context for it.
```

Comment:
Reader is a higher kinded wrapper around a function that goes from `A` to `B`  `(A) -> B`